core/modules/css/KvInterface.h

```diff
@@ -42,18 +42,28 @@ namespace css {
 
 class KvInterface {
 public:
+
+    /**
+     * Sets a max length for kvKey.
+     * If it is changed it must also be changed in the
+     * CssData SQL Schema as well.
+     */
+    const unsigned int kMaxKeyLength = 255;
+
     virtual ~KvInterface() {};
 
     /**
      * Create a key/value pair.
-     * Throws KeyExistsError if the key already exists (or if any other problem,
-     * e.g., a connection error is detected).
+     * key must be shorter than kMaxKeyLength
```

Comment:
Start sentence with capital letter and end with dot. What happens if key length exceeds the limit, should we add special exception for that?